django-stubs/core/mail/message.pyi

```diff
@@ -60,15 +64,17 @@ class EmailMessage:
     def __init__(
         self,
         subject: str = ...,
-        body: Optional[str] = ...,
+        body: str = ...,
         from_email: Optional[str] = ...,
-        to: Optional[Union[Sequence[str], str]] = ...,
-        bcc: Optional[Union[Sequence[str], str]] = ...,
+        to: Optional[Sequence[str]] = ...,
+        bcc: Optional[Sequence[str]] = ...,
         connection: Optional[Any] = ...,
-        attachments: Optional[Union[List[Tuple[str, Union[str, bytes], str]], List[MIMEText]]] = ...,
+        attachments: Sequence[
```

Comment:
Wow! This type looks complex. Let's make an alias.

Reply:
I am talking about: `Union[MIMEBase, Tuple[Optional[Union[MIMEText, str]], Optional[_AttachmentContent], Optional[str]]]`